packages/grafana-ui/src/components/UnitPicker/UnitPicker.story.tsx

```diff
@@ -7,15 +7,25 @@ import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 
 import mdx from './UnitPicker.mdx';
 
+type Props = React.ComponentProps<typeof UnitPicker>;
+
 const meta: ComponentMeta<typeof UnitPicker> = {
   title: 'Pickers and Editors/UnitPicker',
   component: UnitPicker,
   decorators: [withCenteredStory],
   parameters: {
+    controls: {
+      exclude: ['onChange', 'value'],
+    },
     docs: mdx,
   },
 };
 
-export const simple: ComponentStory<typeof UnitPicker> = () => <UnitPicker onChange={(val) => console.log(val)} />;
+export const Basic: ComponentStory<typeof UnitPicker> = (args: Props) => <UnitPicker {...args} />;
+
+Basic.args = {
+  onChange: console.log,
```

Comment:
Instead of the `console.log` lets change this to use the storybook [actions](https://storybook.js.org/docs/react/essentials/actions)

Reply:
Thank you for a great suggestion @JoaoSilvaGrafana, landed in [4a4dc59](https://github.com/grafana/grafana/pull/54694/commits/4a4dc593db5371e09b6d6fcfeb0ab056f61eac0d)